js/util/browser/dom.js

```diff
@@ -51,10 +51,22 @@ exports.suppressClick = function() {
     }, 0);
 };
 
-exports.mousePos = function (el, e) {
+var _cachedOffset;
+function _setOffset(el) {
     var rect = el.getBoundingClientRect();
+    _cachedOffset = new Point(
+        rect.left - el.clientLeft,
+        rect.top - el.clientTop
+    );
+    return _cachedOffset;
+}
+exports.setOffset = function (el) {
+    return _setOffset(el);
+};
+exports.mousePos = function (el, e) {
+    var offset = _cachedOffset || _setOffset(el);
```

Comment:
You can call `exports.setOffset` directly instead of defining an additional function.